bindings-python/gci/componentmodel.py

```diff
@@ -166,6 +170,10 @@ def current_repository_ctx(self):
             return None
         return self.repositoryContexts[-1]
 
+    # XXX rm asap - keep for backards-compat
```

Comment:
if keep it for backwards compatibility, does it mean that you merge the previous localresources and externalResources into resources or do just drop them?

Reply:
I would prefer to keep the field for a transition phase of ~2w (I have to transport this code into our live-ls, unfortunately), then remove it. I think, there is no need to remain backwards-compatible on landscaper-end. I also considered to even (with some monkey-patching) merge those (now legacy) attrs into the new one, but I think this will not be that beneficial

Reply:
ok then I will completely remove the two legacy attribute in the go binding and replace them with the one `resource`.